dioptas/widgets/plot_widgets/HistogramLUTItem.py

```diff
@@ -84,6 +89,18 @@ def __init__(self, image=None, fillHistogram=False, orientation='horizontal', au
         self.gradient = GradientEditorItem()
         self.gradient.loadPreset('grey')
 
+        configurationButton = FlatButton('C')
+        configurationButton.setStyleSheet(
```

Comment:
Style needs to be applied since widget is embedded into a `QGraphicsScene` through a `QGraphicsProxyWidget`.